docs/references/javascript/organization-invitation.mdx

```diff
@@ -15,23 +15,238 @@ The `OrganizationInvitation` object is the model around an organization invitati
+        try {
+          const { totalCount, data } = await organization.getInvitations();
+
+          const invitations = data;
+
+          if (invitations.length === 0) {
+            list.textContent = "No invitations";
+          }
+
+          invitations.map((invitation) => {
+            const li = document.createElement("li");
+            li.textContent = `${invitation.emailAddress} - ${invitation.role}`;
+
+            // Add administrative actions: revoke invitation
+            if (isAdmin) {
+              const revokeBtn = document.createElement("button");
+              revokeBtn.textContent = "Revoke";
+              revokeBtn.addEventListener("click", async function (e) {
+                e.preventDefault();
+                await invitation.revoke();
+              });
+              li.appendChild(revokeBtn);
+            }
+            // Add the entry to the list
+            list.appendChild(li);
+          });
+        } catch (err) {
+          console.error(err);
+        }
+      }
+
+      // Gets the currently active org, checks if the current user
+      // is in the current org and if the user is an admin
+      // and then renders the invitations for the current org.
+      async function userAuthorizationBeforeRenderInvitations() {
+        // Get the currently active organization's ID.
+        const organizationId = clerk.organization.id;
+
+        // Check if the current user is in the currently active organization
+        const organizationMemberships =
+          await clerk.user.getOrganizationMemberships();
+
+        const currentMembership = organizationMemberships.find(
+          (membership) => membership.organization.id === organizationId
+        );
+        const currentOrganization = currentMembership.organization;
+
+        if (!currentOrganization) {
+          return;
+        }
+
+        // Check if the current user is an admin
+        const isAdmin = currentMembership.role === "org:admin";
+
+        renderInvitations(currentOrganization, isAdmin);
+      }
+
+      userAuthorizationBeforeRenderInvitations();
+    } else {
+      document.getElementById("app").innerHTML = `
+        <div id="sign-in"></div>
+      `;
+
+      const signInDiv = document.getElementById("sign-in");
+
+      clerk.mountSignIn(signInDiv);
+    }
+    ```
+  </Tab>
+
+  <Tab>
+    ```html filename="index.html"
+      <div id="app"></div>
+
+      <h2>Invitations List</h2>
+      <ul id="invitations_list"></ul>
+
+      <!-- Initialize Clerk with your
+      Clerk publishable key and Frontend API URL -->
+      <script
+        async
+        crossorigin="anonymous"
+        data-clerk-publishable-key='{{pub_key}}'
+        src="https://[your-domain].clerk.accounts.dev/npm/@clerk/clerk-js@latest/dist/clerk.browser.js"
+        type="text/javascript"
+      ></script>
+
+      <script>
+        window.addEventListener("load", async function () {
+          await Clerk.load();
+
+          if (Clerk.user) {

```

Comment:
This if-else is so big, it's probably better to just check if(!Clerk.user), then mount the component and `return`